USAGE.md

```diff
@@ -126,7 +126,7 @@ aws lambda invoke --function-name GEOSTORE-LAMBDA-FUNCTION-ENDPOINT-NAME \
 
 ```console
 $ aws lambda invoke --function-name datasets \
-    --payload '{"http_method": "POST", "body": {"title": "Auckland_2020"}}' /dev/stdout
+    --payload '{"http_method": "POST", "body": {"title": "Auckland_2020", "description": "Aerial Imagery from APR 2020"}}' /dev/stdout
```

Comment:
Do we want to show an actual response rather than a subset? Here's an example I just ran, which includes several other fields in the response:

```
$ aws lambda invoke --function-name victor-datasets --payload '{"http_method": "POST", "body": {"title": "Auckland_2020", "description": "anything at all"}}' /dev/stdout
{"status_code": 201, "body": {"created_at": "2021-05-26T21:17:47.758448+0000", "pk": "DATASET#01F6N8MSVEY2Y6EPFZ5XR0KFW1", "title": "Auckland_2020", "updated_at": "2021-05-26T21:17:47.758538+0000", "id": "01F6N8MSVEY2Y6EPFZ5XR0KFW1"}}
```